### PR TITLE
[BD-35] PAR-285 feat: change breadcrumb component according to new design

### PR DESCRIPTION
**Pull request by amalbas**
@abutterworth 
Ticket on Jira: https://openedx.atlassian.net/browse/PAR-285
Update breadcrumb component according to new design
breadcrumb-old
breadcrumb-new

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @amalbas! I've created [BLENDED-781](https://openedx.atlassian.net/browse/BLENDED-781) to keep track of it in Jira.


When this pull request is ready, tag your edX technical lead.

File: src/Breadcrumb/index.jsx [hunk 1]
```diff
@@ -3,29 +3,30 @@ import PropTypes from 'prop-types';
...
 const Breadcrumbs = ({
   links, activeLabel, spacer, clickHandler,
 }) => {
   const linkCount = links.length;
 
   return (
-    <nav aria-label="breadcrumb">
-      <ol className={classNames('list-inline')}>
+    <nav aria-label="breadcrumb" className="breadcrumb-container">
+      <ol className={classNames('list-inline', 'd-flex')}>
```
Comment (abutterworth) [inline src/Breadcrumb/index.jsx:15]: Would you mind removing the useless `classNames(~~)` calls in this component? `className="list-linline d-flex"` is easier to read.

File: src/Breadcrumb/index.jsx [hunk 2]
```diff
@@ -3,29 +3,30 @@ import PropTypes from 'prop-types';
...
 
 const Breadcrumbs = ({
   links, activeLabel, spacer, clickHandler,
 }) => {
   const linkCount = links.length;
 
   return (
-    <nav aria-label="breadcrumb">
-      <ol className={classNames('list-inline')}>
+    <nav aria-label="breadcrumb" className="breadcrumb-container">
```
Comment (abutterworth) [inline src/Breadcrumb/index.jsx:14]: Please use the name `pgn__*` for paragon component class names. So in this case `pgn__breadcrumb-container`

File: src/Breadcrumb/index.jsx [hunk 3]
```diff
@@ -3,29 +3,30 @@ import PropTypes from 'prop-types';
...
+      <ol className="list-inline d-flex">
         {links.map(({ url, label }, i) => (
           <React.Fragment key={url}>
             <li className={classNames('list-inline-item')}>
-              <a href={url} {...(clickHandler && { onClick: clickHandler })}>{label}</a>
+              <a className="link-muted" href={url} {...(clickHandler && { onClick: clickHandler })}>{label}</a>
             </li>
             {(activeLabel || ((i + 1) < linkCount))
               && (
               <li className={classNames('list-inline-item')} role="presentation" aria-label="spacer">
```
Comment (abutterworth) [inline src/Breadcrumb/index.jsx:23]:
```suggestion
              <li className="list-inline-item" role="presentation" aria-label="spacer">
```

File: src/Breadcrumb/index.jsx [hunk 4]
```diff
@@ -3,29 +3,30 @@ import PropTypes from 'prop-types';
...
-                {spacer || <Icon className="fa fa-chevron-right" id={`spacer-${i}`} />}
+                {spacer || <Icon src={ChevronRight} id={`spacer-${i}`} />}
               </li>
               )}
           </React.Fragment>
         ))}
-        {activeLabel && <li className={classNames('list-inline-item')} key="active">{activeLabel}</li>}
+        {activeLabel && <li className={classNames('list-inline-item', 'active')} key="active">{activeLabel}</li>}
```
Comment (abutterworth) [inline src/Breadcrumb/index.jsx:29]:
```suggestion
        {activeLabel && <li className="list-inline-item active" key="active">{activeLabel}</li>}
```

**Review by abutterworth — APPROVED** [review]
🐋 🚢

**Pull request merged**